### PR TITLE
Prepare to configure Zookeeper in Marathon

### PR DESCRIPTION
**Pull request by rjmac**

### CODE REVIEW

File: store-pg/docker/secondary.conf.j2 [hunk 1]
```diff
@@ -46,4 +46,15 @@ pg-secondary-config {
...
+  {% if ZOOKEEPER_ENSEMBLE is defined -%}
+  curator = {
+    ensemble = {{ ZOOKEEPER_ENSEMBLE }}
+    session-timeout = 40s
+    connect-timeout = 5s
+    base-retry-wait = 100ms
+    max-retry-wait = 10s
+    max-retries = 29
+  }
+  {%- endif %)
```
Comment (chitang) [inline store-pg/docker/secondary.conf.j2]: Should it be {% and %}?

File: store-pg/docker/secondary.conf.j2 [hunk 2]
```diff
@@ -46,4 +46,15 @@ pg-secondary-config {
   }
 
   tablespace = "{{ PG_SECONDARY_TABLESPACE_FN }}"
+
+  {% if ZOOKEEPER_ENSEMBLE is defined -%}
```
Comment (chitang) [inline store-pg/docker/secondary.conf.j2]: Should there be a - ?

**Comment by rjmac** [comment]
In both those places, the `-` is correct.  See for example [the query-coordinator config file](https://github.com/socrata-platform/query-coordinator/blob/master/query-coordinator/docker/query-coordinator.conf.j2).

**Comment by aerust** [comment]
Seems reasonable to me

**Pull request merged**